ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -61,14 +62,18 @@
 from metadata.ingestion.ometa.ometa_api import OpenMetadata
 from metadata.ingestion.ometa.utils import ometa_logger
 from metadata.orm_profiler.orm.converter import ometa_to_orm
-from metadata.orm_profiler.profiler.default import DefaultProfiler
+from metadata.orm_profiler.profiles.default import DefaultProfiler
```

Comment:
I don't think this should be needed 🤔  I did a rename a few days ago, but the current path is `profiler`: https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/orm_profiler/profiler/default.py.

Please, check your local setup and if this is giving an issue let's take a look 🙏

Reply:
okay I'll check